main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"runtime"
+
+	"github.com/fatih/color"
+	"github.com/ryanuber/columnize"
+
+	"github.com/giantswarm/gsctl/commands"
+)
+
+func init() {
+	columnizeConfig := columnize.DefaultConfig()
+	columnizeConfig.Glue = "   "
+
+	// disable color on windows, as it is super slow
+	if runtime.GOOS == "windows" {
+		color.NoColor = true
+	}
+}
+
+func main() {
+	// top level commands
+	commands.RootCommand.AddCommand(
```

Comment:
+ on internal sub command registration (registering internally with default values)
- on private/unexported commands (having godoc for the command is cool)